.github/workflows/ci.yml

```diff
@@ -66,7 +72,25 @@ jobs:
         env
         # Calculate the short SHA1 hash of the git commit
         echo "::set-output name=short_sha::$(git rev-parse --short HEAD)"
+        echo "::set-output name=closest_tag::$(git describe --abbrev=0)"
+
+    # GoReleaser will take care of publishing those artifacts into the release
+    - name: Run GoReleaser
+      uses: goreleaser/goreleaser-action@v1
+      # if: startsWith(github.ref, 'refs/tags/')
+      with:
+        version: latest
+        args: release --rm-dist --skip-publish
+        # Figure out signing
+        # key: ${{ secrets.YOUR_PRIVATE_KEY }}
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        TAG: ${{ steps.vars.outputs.closest_tag }} #${{ github.ref }}
+
+    - name: Show project tree
+      run: tree
```

Comment:
Need `sudo apt install tree`, also `if: matrix.os == 'ubuntu-latest'`

Reply:
Will only run it on Ubuntu, so I'm splitting the release process into its own yaml.